src/Microsoft.PowerShell.Commands.Utility/commands/utility/AddType.cs

```diff
@@ -568,19 +558,15 @@ private string GetUsingTemplate(Language language)
             {
                 case Language.CSharp:
                     return
-                        "using System;\n" +
-                        "using System.Runtime.InteropServices;\n" +
```

Comment:
https://github.com/PowerShell/PowerShell/pull/6141/files#r171029964
I want to point it out again that this is a breaking change in that `System` and `System.Runtime.InteropServices` are not included when `-UsingNamespace` is not specified after this change.

Reply:
If we add this we get errors:
```
PS C:\Users\sie\Documents\GitHub\iSazonov\PowerShell>         $CSharpCode1 = @"
>> using System;
>> using System.Runtime.InteropServices;
>>         namespace Test.AddType
>>         {
>>             public class CSharpTest1$guid
>>             {
>>                 public static int Add1(int a, int b)
>>                 {
>>                     return (a + b);
>>                 }
>>             }
>>         }
>> "@
PS C:\Users\sie\Documents\GitHub\iSazonov\PowerShell> Add-type -TypeDefinition $CSharpCode1
Add-type : (2,1): hidden CS8019: Unnecessary using directive.
using System.Runtime.InteropServices;
^
At line:1 char:1
+ Add-type -TypeDefinition $CSharpCode1
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ CategoryInfo          : InvalidData: ((2,1): hidden C...sing directive.:CSDiagnostic) [Add-Type], Exception
+ FullyQualifiedErrorId : SOURCE_CODE_ERROR,Microsoft.PowerShell.Commands.AddTypeCommand

Add-type : (1,1): hidden CS8019: Unnecessary using directive.
using System;
^
At line:1 char:1
+ Add-type -TypeDefinition $CSharpCode1
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ CategoryInfo          : InvalidData: ((1,1): hidden C...sing directive.:CSDiagnostic) [Add-Type], Exception
+ FullyQualifiedErrorId : SOURCE_CODE_ERROR,Microsoft.PowerShell.Commands.AddTypeCommand

```

Reply:
Currently Appveyor CI failed with the same error in log line 3827. 😕 

Can I rebase to fix an error with File.wxs?

Reply:
The error seems caused by a warning that more than one `using system;` is specified. I'm sure there is a way to deal with it.

Feel free to rebase.